packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_screen.dart

```diff
@@ -3,9 +3,115 @@
 // found in the LICENSE file.
 
 import 'package:flutter/material.dart';
-
 import '../../shared/screen.dart';
 
+const List<String> paths = [
+  '/shoes/..*',
+  '/Clothes/..*',
+  '/Toys/..*',
+  '/Jewelry/..*',
+  '/Watches/..* ',
+  '/Glasses/..*',
+];
+
+List<LinkData> allLinkDatas = [
+  for (var path in paths)
+    LinkData(
+      os: 'Android, iOS',
+      domain: 'm.shopping.com',
+      path: path,
+      domainError: true,
+      pathError: path.contains('shoe'),
+    ),
+  for (var path in paths)
+    LinkData(
+      os: 'iOS',
+      domain: 'm.french.shopping.com',
+      path: path,
+      pathError: path.contains('shoe'),
+    ),
+  for (var path in paths)
+    LinkData(
+      os: 'Android',
+      domain: 'm.chinese.shopping.com',
+      path: path,
+      pathError: path.contains('shoe'),
+    ),
+];
+
+class LinkData {
+  LinkData({
+    required this.os,
+    required this.domain,
+    required this.path,
+    this.scheme = 'Http://, Https://',
+    this.domainError = false,
+    this.pathError = false,
+  });
+
+  String os;
+  String path;
+  String domain;
+  String scheme;
+  bool domainError;
+  bool pathError;
+  String get searchLabel => (os + path + domain + scheme).toLowerCase();
+
+  DataRow buildRow(
+    BuildContext context, {
+    MaterialStateProperty<Color?>? color,
+  }) {
+    return DataRow(
+      color: color,
+      cells: [
+        DataCell(Text(os)),
+        DataCell(Text(scheme)),
+        DataCell(
+          domainError
+              ? Row(
```

Comment:
instead of using a ternary, you can conditionally add children to the list in Row:

```
Row(
  children: [
    if (domainError)
      Padding(
        padding: const EdgeInsets.only(right: denseSpacing),
        child: Icon (...),
      ),
    Text(domain),
  ],
)
```